src/HighTable.module.css

```diff
@@ -85,18 +91,44 @@
     z-index: 1000;
   }
 
-  /* column menu button */
-  /* TODO: fix the positioning */
+  /* column header controls: default button vs functional controls cluster */
   thead {
     th {
-      & > button {
-        position: absolute;
-        right: 6px;
-        top: 50%;
-        transform: translateY(-50%);
+      /* Default header row layout for non-functional headers */
+      .ht-header-default {
```

Comment:
hmmm, is it possible to override these classes? I haven't use any class in the module, using roles instead.